### PR TITLE
cli: replace short_commit_description() with commit summary template

### PR DESCRIPTION
**Pull request by yuja**
# Checklist

If applicable:
- [ ] I have updated `CHANGELOG.md`
- [ ] I have updated the documentation (README.md, docs/, demos/)
- [ ] I have added tests to cover my changes

### CODE REVIEW

File: src/commands.rs [hunk 1]
```diff
@@ -3734,6 +3702,9 @@ fn cmd_workspace_list(
     for (workspace_id, checkout_id) in repo.view().wc_commit_ids().iter().sorted() {
         write!(ui, "{}: ", workspace_id.as_str())?;
         let commit = repo.store().get_commit(checkout_id)?;
+        // TODO: Should the "current" workspace be the one user operates on or the one
+        // currently listing? Here workspace_id != workspace_command.workspace_id(), and
+        // that's why workspace_command.write_commit_summary() isn't used.
```
Comment (martinvonz) [inline src/commands.rs:3707]: Good question. I'd say it should be the current workspace, so the current behavior is incorrect. What do you think?

**Comment by martinvonz** [comment]
> `start_transaction(&'a mut self) -> Wrapper<'a>`

FWIW, I like that idea.

**Review by martinvonz — APPROVED** [review]
Thanks for cleaning up!

File: src/commands.rs (hunk 1 above)
Comment (yuja) [reply src/commands.rs:3707]: Yeah, it's more consistent with the other commands, and simpler. I'll fix that.

**Pull request merged**